Source/UINavigation/Private/UINavPCComponent.cpp

```diff
@@ -260,13 +260,15 @@ void UUINavPCComponent::SetActiveWidget(UUINavWidget * NewActiveWidget)
 		if (NewActiveWidget == nullptr)
 		{
 			IUINavPCReceiver::Execute_OnRootWidgetRemoved(GetOwner());
-			if (IsValid(PC) && IsValid(PC->Player))
+			// GetOwningPlayer will not be valid if SetActiveWidget is called during PlayerController EndPlay()
+			APlayerController *ActiveWidgetPC = ActiveWidget->GetOwningPlayer(); 
+			if (IsValid(ActiveWidgetPC) && ActiveWidgetPC->IsLocalPlayerController() && IsValid(ActiveWidgetPC->Player))
```

Comment:
You don't need to check for if it's a local player controller. This should only be called on the client.